articles/azure-arc/kubernetes/use-gitops-connected-cluster.md

```diff
@@ -178,49 +218,62 @@ Command group 'k8sconfiguration' is in preview. It may be changed/removed in a f
+    "createdBy": null,
+    "createdByType": null,
+    "lastModifiedAt": "2020-12-10T18:30:56.881219+00:00",
+    "lastModifiedBy": null,
+    "lastModifiedByType": null
+},
   "type": "Microsoft.KubernetesConfiguration/sourceControlConfigurations"
 }
 ```
 
 When the `sourceControlConfiguration` is created, a few things happen under the hood:
 
-1. The Azure Arc `config-agent` monitors Azure Resource Manager for new or updated configurations (`Microsoft.KubernetesConfiguration/sourceControlConfiguration`)
+1. The Azure Arc `config-agent` monitors Azure Resource Manager for new or updated configurations (`Microsoft.KubernetesConfiguration/sourceControlConfigurations`)
 1. `config-agent` notices the new `Pending` configuration
 1. `config-agent` reads the configuration properties and prepares to deploy a managed instance of `flux`
     * `config-agent` creates the destination namespace
     * `config-agent` prepares a Kubernetes Service Account with the appropriate permission (`cluster` or `namespace` scope)
     * `config-agent` deploys an instance of `flux`
-    * `flux` generates an SSH key and logs the public key
-1. `config-agent` reports status back to the `sourceControlConfiguration`
+    * `flux` generates an SSH key and logs the public key (if using the option of SSH with Flux-generated keys)
+1. `config-agent` reports status back to the `sourceControlConfiguration` resource in Azure
 
 While the provisioning process happens, the `sourceControlConfiguration` will move through a few state changes. Monitor progress with the `az k8sconfiguration show ...` command above:
 
 1. `complianceStatus` -> `Pending`: represents the initial and in-progress states
 1. `complianceStatus` -> `Installed`: `config-agent` was able to successfully configure the cluster and deploy `flux` without error
 1. `complianceStatus` -> `Failed`: `config-agent` encountered an error deploying `flux`, details should be available in `complianceStatus.message` response body
 
-## Apply configuration from a private git repository
+## Apply configuration from a private Git repository
+
+If you are using a private Git repo, then you need to configure the SSH public key in your repo. You can configure the public key either on the Git repo or the Git user that has access to the repo. The SSH public key will be either the one you provide or the one that Flux generates.
+
+**Get your own public key**
 
-If you are using a private git repo, then you need to perform one more task to close the loop: add the public key generated by `flux` as a **Deploy key** in the repo.
+If you generated your own SSH keys, then you already have the private and public keys.
 
-**Get the public key using Azure CLI**
+**Get the public key using Azure CLI (useful if Flux generates the keys)**
 
 ```console
 $ az k8sconfiguration show --resource-group <resource group name> --cluster-name <connected cluster name> --name <configuration name> --query 'repositoryPublicKey'
```

Comment:
This command is missing a required parameter **--cluster-type connectedClusters**